app/src/main/java/ro/code4/deurgenta/bindings/BindingAdapters.kt

```diff
@@ -0,0 +1,30 @@
+package ro.code4.deurgenta.bindings
+
+import android.annotation.SuppressLint
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import com.bumptech.glide.Glide
+
+object BindingAdapters {
+    @SuppressLint("ResourceType")
+    @JvmStatic
+    @BindingAdapter(value = ["imageRes"], requireAll = false)
+    fun bindImageResource(imageView: ImageView, resId: Int?) {
```

Comment:
where is this used?

Reply:
Correct, not used atm. It was used but then I decided to do it in the next PR. Will remove it.